lib/beaker/host/pswindows.rb

```diff
@@ -23,5 +23,17 @@ def platform_defaults
       })
     end
 
+    attr_reader :scp_separator, :external_copy_base, :system_temp_path
+    def initialize name, host_hash, options
+      super
+
+      @scp_separator = '/'
+      # %TEMP% == C:\Users\ADMINI~1\AppData\Local\Temp
+      # is a user temp path, not the system path.  Also, it doesn't work, there's
+      # probably an issue with the `ADMINI~1` section
+      @system_temp_path = 'C:\\Windows\\Temp'
+      @external_copy_base = '/programdata'
```

Comment:
If you're going to use this for Cygwin:

```ruby
@external_copy_base = execute('echo `cygpath -smF 35`/')
```

You should use this here:

```ruby
@external_copy_base = execute('cmd.exe /c "for %I in (%ALLUSERSPROFILE%) do @echo %~sI"')
```

FWIW, `\ProgramData` is not valid on Windows 2003